modules/monitoring/views/extinfo/components/operating.php

```diff
@@ -1,11 +1,12 @@
 <div class="information-component">
-	<div class="information-component-title">Operating status</div>
+		<div class="information-component-title">
```

Comment:
These lines seem to use a larger indentation compared to the rest of the file. Also indent the lines after the `information-component` div once.

Reply:
Fixed. At least in my IDE it looks good.